main.go

```diff
@@ -84,20 +87,26 @@ func main() {
 
 	if opts.URI == "" {
 		opts.URI = os.Getenv("MONGODB_URI")
```

Comment:
kong let's to override value by using `env` tag https://github.com/alecthomas/kong#supported-tags.